risks/MASVS-STORAGE/1-store-sensitive-data-securely/data-unencrypted-external/android-data-unencrypted-external-static/test.md

```diff
@@ -0,0 +1,25 @@
+---
+platform: android
+title: Usage of API to get to external storage locations
```

Comment:
Recently, we decided to use `Shared Storage Requiring No User Interaction` instead of `External Storage`. In this case, shouldn't the title be `Use of APIs to access Shared Storage Locations Requiring No User Interaction`?

Reply:
@serek8 that'd apply to the `risk.md` which is the most generic, the test are very specific, in this case we're testing "android external storage APIs" specifically, right?

Reply:
I updates the `test` and `risk` accordingly